config/cp_mode.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
de we want to create a subfolder b1/

Reply:
@po09i What are your thoughts about this ? I recall at some point we were talking about having b1/b0 folders but I guess both `b1shim` and `load_vop` could go directly in `shimmingtoolbox` (I even think about putting `load_vop` as a simple function in `b1shim`)

Reply:
Moved this conversation to #208